includes/notes/class-wc-admin-notes-welcome-message.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * WooCommerce Admin: Welcome to WooCommerce Admin Note
+ *
+ * Adds welcome notes to store alerts area.
+ *
+ * @package WooCommerce Admin
+ */
+
+defined( 'ABSPATH' ) || exit;
+/**
+ * WC_Admin_Notes_Welcome_Message.
+ */
+class WC_Admin_Notes_Welcome_Message {
+	const NOTE_NAME = 'woocommerce-admin-welcome-note';
+
+	/**
+	 * Creates a note for welcome message.
+	 */
+	public static function add_welcome_note() {
+
```

Comment:
I think we should add [this check](https://github.com/woocommerce/woocommerce-admin/blob/master/includes/notes/class-wc-admin-notes-historical-data.php#L22-L25) so we make sure we don't create this note when upgrading the plugin:
```suggestion
		$is_upgrading = get_option( WC_Admin_Install::VERSION_OPTION );
		if ( $is_upgrading ) {
			return;
		}
```

Reply:
Thanks for reminding me. I have implemented the changes and added this check.